test/BUILD.gn

```diff
@@ -149,6 +150,9 @@ test("brave_unit_tests") {
 
   if (brave_rewards_enabled) {
     deps += [
+      "//brave/vendor/bat-native-usermodel",
+      "//brave/vendor/bat-native-ads",
```

Comment:
actually I'm confused about why we have all of these if bat-native-ads and bat-native-confirmations are deps of bat-native-ledger